docker/README.md

```diff
@@ -17,14 +17,14 @@ Note: By default the container uses the `pre-cxx11-abi` version of Torch + Torch
 
 ### Instructions
 
-- The example below uses CUDNN 8.9 and TensorRT 8.6
+- The example below uses TensorRT 8.6
```

Comment:
Just FYI (you don't need to change here but I changed this to TRT 10 in my docker PR).